application/settings.py

```diff
@@ -37,15 +37,15 @@
     'django.contrib.contenttypes',
     'django.contrib.staticfiles',
     'django.contrib.gis',
+    'django.contrib.auth',
```

Comment:
Why do you need `Permission`? What, in adding the geojson data, made a change that requires permission objects?